### PR TITLE
Store decimals in standard notation #9577

### PR DESCRIPTION
**Pull request by phebal**
When emitting XML store decimal type values in floating-point notation.

### CODE REVIEW

File: spec/ncs_navigator/warehouse/models/mdes_model_spec.rb [hunk 1]
```diff
@@ -33,9 +33,10 @@ class Spec::Sample::GenerationalTableau
         property   :color_scale, NcsNavigator::Warehouse::DataMapper::NcsString,
                    :pii => true,
                    :set => %w(3 4 5 6 7 8 -6)
+        property   :sample_size, NcsNavigator::Warehouse::DataMapper::NcsDecimal
```
Comment (hannahwhy) [inline spec/ncs_navigator/warehouse/models/mdes_model_spec.rb:36]: Why was a new property added?

If this is required for an import, it should be in a separate commit.

Comment (phebal) [inline spec/ncs_navigator/warehouse/models/mdes_model_spec.rb:36]: It's added to test decimal extraction during emit-xml.  I think it's pretty clear from the specs.
